app/views/dashboard/index.html.erb

```diff
@@ -96,16 +96,14 @@
   <strong>Walltaker is an app built by adults, for adults. You must be over the age of 18 to use it.</strong>
   Walltaker is
   inspired by the <a href="https://www.wallclaimer.com/">WallClaimer</a> app, which allows you to set the
-  wallpaper of your friends phones. This however leads to some weird cases where people post stuff that'd be on
-  your
-  metaphorical blacklist. With that in mind, I wanted it to be restricted to
+  wallpaper of your friends phones. This however leads to some weird cases where people post stuff that'd be on your metaphorical blacklist. With that in mind, I wanted it to be restricted to
   <a href="https://e621.net">e621.net</a>
   results, with an enforced blacklist. This keeps you in control, but not <em>too much</em> control.
 </p>
 
 <h2 id="what-do-i-need">What do I need?</h2>
 <p>
-  Walltaker requires you to <strong>make an account on the website (here!) first.</strong> Once you have an
+  Walltaker requires you to <strong>make an account on the website (<a href="/signup" target="_blank">here!</a>) first.</strong> Once you have an
```

Comment:
Sorry! But would you be able to use the erb helpers instead? They can update if routes change.

```suggestion
  Walltaker requires you to <strong>make an account on the website (<%= link_to 'here!', signup_path, target: '_blank %>) first.</strong> Once you have an
```

(Please verify first, working from my phone)